specs/notation-cli.md

```diff
@@ -36,6 +37,7 @@ COMMANDS:
    certificate, cert  Manage certificates used for verification
    key                Manage keys used for signing
    list, ls           List signatures from remote
+   login              Provide credentials for authenticated registry operations   
```

Comment:
```suggestion
   login              Login to a registry with credentials for authenticated registry operations
```

Reply:
Saying login, where the parameter says login feels redundant. I was just trying to find a synonym verb.